hugo.toml

```diff
@@ -2,4 +2,4 @@ baseURL = "https://example.com/"
 
 [module]
 [module.hugoVersion]
-min = "0.77.0"
\ No newline at end of file
+min = "0.124.0"
```

Comment:
I need to check if this file is really needed. I think not.
Do you know anything about it?

Reply:
Anyway, could you change the `min_version` value from `theme.toml` too?

Reply:
> Anyway, could you change the `min_version` value from `theme.toml` too?

Changed now.